README.md

```diff
@@ -20,7 +20,7 @@ Maven
 <dependency>
     <groupId>com.sinch</groupId>
     <artifactId>sdk-sms</artifactId>
-    <version>1.1.6</version>
+    <version>1.1.7</version>
```

Comment:
This is an internal release so we should not update this. The external will have a different version